woocommerce-yotpo/wc_yotpo.php

```diff
@@ -388,17 +419,18 @@ function wc_yotpo_conversion_track($order_id) {
 }
 
 function wc_yotpo_get_degault_settings() {
-	return array( 'app_key' => '',
-				  'secret' => '',
-				  'widget_location' => 'footer',
-				  'language_code' => 'en',
-				  'widget_tab_name' => 'Reviews',
-				  'bottom_line_enabled_product' => true,
-				  'bottom_line_enabled_category' => false,
-				  'yotpo_language_as_site' => true,
-				  'show_submit_past_orders' => true,
-				  'disable_native_review_system' => true,
-				  'native_star_ratings_enabled' => 'no');
+    return array('app_key' => '',
```

Comment:
@sa3iedsabbah can the indentation be on the same vertical line (including 'app_key')?

Reply:
The indentation is correct if you open it in IDE, it must be the GitHub which is displaying it incorrectly.